modules/ob-about-builds.adoc

```diff
@@ -0,0 +1,28 @@
+// Ths module is included in the following assembly:
+//
+// * builds/overview-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="about-builds_{context}"]
+= About build
+
+The `Build` resource defines the source code of your application and the location where your application images will be pushed. The following example shows a simple build that consists of a git source, a build strategy, and an output image:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: kaniko-golang-build
+spec:
+  source:
+    git:
+      url: https://github.com/username/taxi
+  strategy:
+    name: kaniko
+    kind: ClusterBuildStrategy
+  output:
+    image: registry.mycompany.com/my-org/taxi-app:latest
+----
+
+You can also extend a `Build` resource to push your images to a private registry or use a podman file. 
```

Comment:
For a general reference to the Podman tool:

```suggestion
You can also extend a `Build` resource to push your images to a private registry or use a Podman file. 
```

or for a reference to the `podman` command itself:

```suggestion
You can also extend a `Build` resource to push your images to a private registry or use a `podman` file. 
```

However, per the `value: /tekton/home/.podman` reference in the strategies section, I'm wondering if this should this be:

```
a `.podman` file.
```
?

Reply:
yes, the refernce should be a `.podman` file. Thanks